libs/libnvme/Makefile

```
@@ -0,0 +1,55 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+# Copyright (C) 2023 Luca Barbato
+
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/kernel.mk
+
+PKG_NAME:=libnvme
+PKG_VERSION:=1.5
+PKG_RELEASE:=1
+
+PKG_SOURCE:=libnvme-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/linux-nvme/$(PKG_NAME)/archive/refs/tags/v$(PKG_VERSION)
+PKG_BUILD_DIR:=$(BUILD_DIR)/libnvme-$(PKG_VERSION)
+PKG_HASH:=f73ba1edde059b2d5e7c1048ad4f895e6047bff241c94b34a7aff5894779d086
```

```suggestion
PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
PKG_SOURCE_URL:=https://github.com/linux-nvme/libnvme/archive/refs/tags/v$(PKG_VERSION)
PKG_HASH:=f73ba1edde059b2d5e7c1048ad4f895e6047bff241c94b34a7aff5894779d086
```

PKG_BUILD_DIR defaults to `$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)`, no need to specify